freud/cluster.pyx

```diff
@@ -99,7 +99,7 @@ cdef class Cluster:
                 Simulation box (Default value = None).
         """
         points = freud.common.convert_array(
-            points, 2, dtype=np.float32, contiguous=True)
+            points, 2, dtype=np.float32)
```

Comment:
Remove dtype (it defaults to float32). Also please combine lines 101-102 (and similarly elsewhere in this PR, if the function can be written on one line without hitting flake8 line length limits).